src/libraries/Native/Unix/System.Native/pal_random.c

```diff
@@ -41,7 +41,7 @@ void SystemNative_GetNonCryptographicallySecureRandomBytes(uint8_t* buffer, int3
             do
             {
 #if HAVE_O_CLOEXEC
-                fd = open("/dev/urandom", O_RDONLY, O_CLOEXEC);
```

Comment:
Do we need the same fix here https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/coreclr/src/pal/src/misc/miscpalapi.cpp#L248 as well?

Reply:
Yes. Force-pushing with this fix as well.  (I really wish we had Coccinelle for C++ and not only C.  Would be easier to find these.)